test/feeds.test.js

```diff
@@ -62,3 +62,54 @@ describe('test /feeds/:id responses', () => {
     expect(res.body instanceof Array).toBe(false);
   });
 });
+
+describe('test POST /feeds endpoint', () => {
+  it('responds with json', async () => {
+    const feedData1 = {
```

Comment:
Minor nit: all these variables can be named `feedData` without the 1, 2, 3 suffix, since they are local to the function.